py/desiutil/dust.py

```diff
@@ -189,7 +190,8 @@ def __init__(self, mapdir=None, north="SFD_dust_4096_ngp.fits",
                  south="SFD_dust_4096_sgp.fits", scaling=1.):
 
         if mapdir is None:
-            mapdir = os.environ.get('DUST_DIR', '')
+            dustdir = os.environ.get('DUST_DIR', '')
```

Comment:
So if `DUST_DIR` is not set, `dustdir = ''`, and `mapdir = '/maps'`.  I think it would be better to raise an exception saying that the maps can't be found, rather than wait until a somewhat obscure error is raised when looking for fits files in a directory that doesn't exist.